### PR TITLE
Fix nightly release page link in bottom bar

### LINKED ISSUE #4713: Fix nightly release page link in bottom bar
From @JordanNoone 

Image

### PR DESCRIPTION
**Pull request by pierremtb**
Fixes #4713

Quick change to unbreak the release link on nightly builds. Tested locally. To reproduce:

```
yarn files:flip-to-nightly
yarn tron:start
```

Will put the CI changes to have the notes dumped there in another PR, this is just a quick fix to get started.

### CODE REVIEW

File: src/components/Settings/AllSettingsFields.tsx [hunk 1]
```diff
@@ -271,7 +271,7 @@ export const AllSettingsFields = forwardRef(
               , and start a discussion if you don't see it! Your feedback will
               help us prioritize what to build next.
             </p>
-            {PACKAGE_NAME.indexOf('-nightly') === -1 && (
```
Comment (nadr0) [inline src/components/Settings/AllSettingsFields.tsx:274]: Talked to Pierre, this logic is now backwards. We need `!IS_NIGHTLY`

File: src/components/Settings/AllSettingsFields.tsx [hunk 2]
```diff
@@ -271,7 +271,7 @@ export const AllSettingsFields = forwardRef(
...
+            {IS_NIGHTLY && (
```
Comment (pierremtb) [inline src/components/Settings/AllSettingsFields.tsx:274]: As pointed out by @nadr0 this needs to be

```suggestion
            {!IS_NIGHTLY && (
```

**Review by nadr0 — APPROVED** [review]
Thanks for fixing that one liner bug. Looks good. The URL formatter looks good.

File: src/components/Settings/AllSettingsFields.tsx [hunk 3]
```diff
@@ -246,10 +246,8 @@ export const AllSettingsFields = forwardRef(
                   to inject the version from package.json */}
               App version {APP_VERSION}.{' '}
               <a
-                onClick={openExternalBrowserIfDesktop(
-                  `https://github.com/KittyCAD/modeling-app/releases/tag/v${APP_VERSION}`
-                )}
-                href={`https://github.com/KittyCAD/modeling-app/releases/tag/v${APP_VERSION}`}
+                onClick={openExternalBrowserIfDesktop(RELEASE_URL)}
+                href={RELEASE_URL}
```
Comment (pierremtb) [inline src/components/Settings/AllSettingsFields.tsx:250]: @nadr0 Made the change here that we talked about on Slack. There are a few other places in the app that could benefit from the change but I don't want to blow up the scope of this quick one before the next nightly without being able to go through more extensive testing. I was able to test this locally, both for nightly and for regular.

Comment (nadr0) [reply src/components/Settings/AllSettingsFields.tsx:250]: Thanks we can merge it!

**Pull request merged**